### ISSUE TITLE
Add Npm to 'Path' of NodeTask

### ISSUE THREAD

**Issue by cdietrich** [srs/gradle-node-plugin#134]
Running on a machine with no node/npm installed

When i execute a NodeTask calling a Module that calls Npm via exec then it wont find npm / use the destroyed symlink extracted. thus it would be nice if this would work out of the box instead of using

```
task npmInstallNpm(type: NpmTask, dependsOn: npmSetup) {
    group 'Node'
    description 'Installs NPM"'
    args = [ 'install', '-g', 'npm' ]
}
```

here is what the node module `vsce` does (one of may npm calls)

```
require('child_process').exec('npm -v' ....)
```

**Comment by Hildebrand** [comment]
This is indeed an issue. The yeoman doctor (part of the yo NPM package) also does a version check that fails if being ran from the gradle-node locally installed Node. To be precise, it's this file: https://github.com/yeoman/doctor/blob/master/lib/rules/npm-version.js failing to find npm.

as I have npm installed globally already I wonder if @cdietrich solution will help..

**Comment by cbornet** [comment]
:+1: I also have the yo package installation failure.

**Comment by cbornet** [comment]
To reproduce very easily:
```
./gradlew npm_install_yo
```

**Comment by cbornet** [comment]
Looks like the same issue as #146 introduced by yarn support in 0.14
Downgrading to 0.13 fixes it (but you lose yarn support :disappointed: )

**Comment by s0x** [comment]
This is related to #164 and would be fixed by MR #165
